src/utils.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * utils.ts
+ * Copyright: Microsoft 2019
+ */
+import map from 'lodash/map';
```

Comment:
I think this should still come from the lodash mini unless we change all lodash mini imports
(@deregtd thoughts?)